65.md

```diff
@@ -4,73 +4,58 @@ NIP-65
+  - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience.
+  - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
+## Final Considerations
 
-### Suggestions
+1. Clients SHOULD guide users to keep `kind:10002` lists small (2-4 relays). 
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+2. Clients SHOULD spread an author's `kind:10002` events to as many relays as viable. 
 
-Authors may post events outside of the feed that they wish their followers to follow by posting them to relays outside of those listed in their "Relay List Metadata".  For example, an author may want to reply to someone without all of their followers watching.
+3. `kind:10002` events MUST only be used to advertise the user's preferred relays to others. The Client's configuration SHOULD stay in other lists where other factors can be considered, such as specific relays for a given UI feature of the Client.
 
-It is suggested that relays allow any user to write their own kind `10002` event (optionally with AUTH to verify it is their own) even if they are not otherwise subscribed to the relay because
+4. DMs SHOULD only be broadcasted to the author's WRITE relays and to the receiver's READ relays to keep maximum privacy. 
 
-  - finding where someone posts is rather important
-  - these events do not have `content` that needs management
-  - relays only need to store one replaceable event per pubkey to offer this service
-
-### Why not in kind `0` Metadata
-
-Even though this is user related metadata, it is a separate event from kind `0` in order to keep it small (as it should be widely spread) and to not have `content` that may require moderation by relay operators so that it is more acceptable to relays.
-
-### Example
-
-```json
-{
-  "kind": 10002,
-  "tags": [
-    ["r", "wss://alicerelay.example.com"],
-    ["r", "wss://brando-relay.com"],
-    ["r", "wss://expensive-relay.example2.com", "write"],
-    ["r", "wss://nostr-relay.example.com", "read"],
-  ],
-  "content": "",
-  ...other fields
-```
+5. Relays that support this NIP MUST accept and serve kind 10002 events for everybody whether they are paying customers or not.
```

Comment:
Is this line necessary? I think relay owners shouldn't be forced to accept a load of spam just because it has kind 10002 and they probably won't accept it, so better not have this line here as it may cause more confusion than help.

Reply:
I was fine with it because of the IF. IF the relay declares support for NIP-65, it MUST accept it. Relays can choose to not declare support.

Reply:
This nip used to explain to relay operators why this isn't too big of a deal, but that disappeared from Vitor's rewrite:

- finding where someone posts is rather important
- these events do not have content that needs management
- relays only need to store one replaceable event per pubkey to offer this service

If someone tries to store a kind-10002 that has any content at all, relays are free to (and I suggest that they do) reject such events.